js/rollup.config.mjs

```diff
@@ -66,11 +66,5 @@ export default [
       // List any external dependencies here
     ]
   },
-  ...generateBundleAndTypeBundles('index'),
-  ...generateBundleAndTypeBundles('frameworks/cloudflare'),
-  ...generateBundleAndTypeBundles('frameworks/langchain'),
-  ...generateBundleAndTypeBundles('frameworks/openai'),
-  ...generateBundleAndTypeBundles('frameworks/langgraph'),
-  ...generateBundleAndTypeBundles('frameworks/vercel'),
-  ...generateBundleAndTypeBundles('sdk/index'),
+  ...generateBundleAndTypeBundles('index')
```

Comment:
The removal of framework bundles (cloudflare, langchain, openai, etc.) is a breaking change that should be documented. Consider adding a migration guide for users who depend on these framework-specific bundles. Also, this change should be reflected in the version bump with a major version increase following semver.